WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletSuccessViewModel.cs

```diff
@@ -1,20 +1,46 @@
-﻿using NBitcoin;
+﻿using System;
+using NBitcoin;
 using ReactiveUI;
 using WalletWasabi.Gui.ViewModels;
+using WalletWasabi.Helpers;
+using WalletWasabi.KeyManagement;
+using WalletWasabi.Logging;
 
 namespace WalletWasabi.Gui.Tabs.WalletManager
 {
 	internal class GenerateWalletSuccessViewModel : CategoryViewModel
 	{
 		private string _mnemonicWords;
+		private string _password;
+		private string _validationMessage;
 
-		public GenerateWalletSuccessViewModel(WalletManagerViewModel owner, Mnemonic mnemonic) : base("Wallet Generated Successfully!")
+
+		public GenerateWalletSuccessViewModel(WalletManagerViewModel owner, Mnemonic mnemonic, BitcoinEncryptedSecretNoEC encryptedSecret, string password, string walletFile) 
+			: base("Wallet Generated Successfully!")
 		{
 			_mnemonicWords = mnemonic.ToString();
 
 			ConfirmCommand = ReactiveCommand.Create(() =>
 			{
-				owner.SelectLoadWallet();
+				Password = Guard.Correct(Password); // Don't let whitespaces to the beginning and to the end.
+
+				if (password != Password)
+				{
+					ValidationMessage = "Password doesn't match";
+					return;
+				}
+
+				try
+				{
+					encryptedSecret.GetSecret(Password + "trolo");
```

Comment:
only for debug right?

Reply:
yes. This PR was pushed for discussing the pros/cons of the idea with something real (with code) and to get feedback, i was never for merging. Anyway, if this is good then I will remove this failing trick.